.env.demo.compose

```
@@ -565,3 +565,42 @@ MCP_WS_SESSION_COOKIE_NAME=mcp-ws-session-id
 MCP_WS_SESSION_ENABLED=true
 MCP_WS_TLS=false
 MCP_WS_TRUSTED_PROXIES=
+
+# OAuth 2.0 Authorization Configuration (Demo Environment)
+MCP_AUTH_SERVERS=https://authdemo.gauzy.co
+MCP_AUTH_ENABLED=true
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The MCP_AUTH_ENABLED key should go before the MCP_AUTH_SERVERS key

(UnorderedKey)